### DISCUSSION TITLE
Can I remap a key to <dead_greek> with this tool?

### DISCUSSION THREAD

**Discussion by caiigee** [xremap/xremap#559] [Q&A]
Hi everyone, recently I discovered the [Compose sequences](https://cgit.freedesktop.org/xorg/lib/libX11/plain/nls/en_US.UTF-8/Compose.pre) and decided to enable the Compose key on my Hyprland setup. However, because I am frequently writing about science topics, being able to write greek letters on the fly would be amazing.

So I started researching about how I could possibly remap my `lvl3 w` (AltGr+w) to the `<dead_greek>` key and I came across this tool. Because I am using Hyprland on NixOS, a program such as `xmodmap` will not work, so can I somehow achieve my goal with this program?

**Comment by plundration** [comment]
Would love to know also.

**Comment by k0kubun** [comment]
Please run xremap with `RUST_LOG=debug xremap ...`, input the sequence using a keyboard that can emit it, and look at the output of the xremap process. You should see the key events you need to emit.
